src-tauri/src/system_tray.rs

```diff
@@ -1,22 +1,30 @@
-use crate::app_url;
+use crate::{app_url, executor_url};
 
 use crate::config::log_path;
 use tauri::{
     AppHandle, CustomMenuItem, Manager, SystemTray, SystemTrayMenu, SystemTrayMenuItem,
     WindowBuilder, WindowUrl, Wry,
 };
 use std::fs;
-use directories::UserDirs;
+use crate::logs::open_logs_folder;
+use crate::util::open_url;
+use tauri::ClipboardManager;
 
 pub fn build_system_tray() -> SystemTray {
     let show_ad4min = CustomMenuItem::new("show_ad4min".to_string(), "Show Ad4min");
     let quit = CustomMenuItem::new("quit".to_string(), "Quit");
-    let copy_logs = CustomMenuItem::new("copy_logs".to_string(), "Copy Logs");
+    let open_logs = CustomMenuItem::new("open_logs".to_string(), "Open Logs");
```

Comment:
copy_url seems not useful as a system tray menu item.
open_logs and report_issue should be a menu item (help -> open logs, report issue), not a system tray item.